<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YY_CHATClass</class>
 <widget class="QMainWindow" name="YY_CHATClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>228</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YY_CHAT</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0" colspan="3">
     <widget class="QToolBox" name="toolBox">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_Chat">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>210</width>
         <height>326</height>
        </rect>
       </property>
       <attribute name="label">
        <string>YY聊天</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTreeWidget" name="treeWidget_Chat">
          <property name="autoExpandDelay">
           <number>-1</number>
          </property>
          <column>
           <property name="text">
            <string>IP</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>姓名</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>格言</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>210</width>
         <height>326</height>
        </rect>
       </property>
       <attribute name="label">
        <string>YY群聊</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QTreeWidget" name="treeWidget_ChatGroup">
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_Calendar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>338</width>
         <height>310</height>
        </rect>
       </property>
       <attribute name="label">
        <string>日历日期</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QCalendarWidget" name="calendarWidget"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_Note"/>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_SetCenter">
      <property name="text">
       <string>设置中心</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="yy_chat.qrc"/>
 </resources>
 <connections/>
</ui>
